examples/tests/invalid/bypass_return.test.w

```diff
@@ -0,0 +1,20 @@
+let y: (): num = () => {
+    return "a";
+//         ^^^ Expected type to be "num", but got "str" instead
+};
+
+let wrongReturnInScope = (): num => {
+  if false {
+    return "a";
+         //^^^ Expected type to be "num", but got "str" instead
+  }
+
+  throw "error";
+}
+
+let missingReturn = (): num => {
+  if true {
+    throw "error";
+  }
+}
+//^ Expected to return "num"
```

Comment:
Can we add a test case where the function body is empty? I believe it should still raise an error in this scenario